package.json

```
@@ -1,10 +1,11 @@
 {
   "name": "@taikai/locales",
-  "version": "1.0.0-alpha.4",
+  "version": "1.0.0-alpha.4.1",
   "description": "Expose the locales for the Taikai project",
   "main": "dist/lib/index.js",
   "types": "dist/lib/index.d.ts",
   "scripts": {
+    "generate-frontend-types": "node ./scripts/generate-frontend-types.js && npm run format",
```

```suggestion
    "generateTypes": "node ./scripts/generate-frontend-types.js && npm run format",
```

We tend to use the same command in the `CMS and Frontend`, and my impression is that it improves DX if the same command with the same intent is replicated across repositories.